aspnetcore/blazor/fundamentals/routing.md

```diff
@@ -704,6 +704,31 @@ The following component:
 
 For more information on component disposal, see <xref:blazor/components/component-disposal>.
 
+:::moniker range=">= aspnetcore-9.0"
+
+## Navigation Manager redirect behavior during static server-side rendering (static SSR)
+
+For a redirect during static server-side rendering (static SSR), <xref:Microsoft.AspNetCore.Components.NavigationManager> relies on throwing a <xref:Microsoft.AspNetCore.Components.NavigationException> that gets captured by the framework, which converts the error into a redirect. Code that exists after the call to <xref:Microsoft.AspNetCore.Components.NavigationManager.NavigateTo%2A> isn't called. When using Visual Studio, the debugger breaks on the exception, requiring you to deselect the checkbox for **Break when this exception type is user-handled** in the Visual Studio UI to avoid the debugger stopping for future redirects.
+
+:::moniker-end
+
+:::moniker range=">= aspnetcore-10.0"
+
+You can use the `<BlazorDisableThrowNavigationException>` MSBuild property set to `true` in the app's project file to opt-in to no longer throwing a <xref:Microsoft.AspNetCore.Components.NavigationException>. This behavior is enabled by default in the .NET 10 or later Blazor Web App project template:
```

Comment:
It's probably also worth mentioning that this means code after the navigation will now run when it wouldn't have before.